tests/virt_control_plane_test.go

```diff
@@ -299,6 +299,10 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 
 				By("ensuring we now have a ready virt-handler daemonset")
 				Eventually(readyFunc, 30*time.Second, time.Second).Should(BeNumerically("==", desiredDeamonsSetCount))
+
+				By("ensuring that the config update watcher eventually resumes the connection")
+				kv := util.GetCurrentKv(virtCli)
+				tests.WaitForConfigToBePropagatedToComponent("kubevirt.io=virt-handler", kv.ResourceVersion, tests.ExpectResourceVersionToBeLessThanConfigVersion, 40*time.Second)
```

Comment:
Wouldn't it be better to update the KV and wait for the new version?

Reply:
@xpivarc sorry if I miss something here, but don't we wait for the new version using `tests.ExpectResourceVersionToBeLessThanConfigVersion`?

Reply:
I think it would work as is due to status updates, but doing a random explicit change to really ensure that a config update needs to happen makes a lot of sense. :+1:  Done.

Reply:
Thanks Roman. @enp0s3 yes but we don't make an explicit change.

Reply:
I think it's redundant to call the `Wait*` now.

Reply:
@xpivarc I don't think so. Waiting for the config to be finally picked up is the prove that we recover correctly. It can however take some time due to accumulated back-off delays.

Reply:
@rmohr @xpivarc IIUC by updating the KV CR you want to enforce the config watcher on virt-handler to get out from the back-off timeout, the event handler of the update should be called in virt-handler, thus updating the lastValidResourceVersion in the clusterConfig. Actually now I have a concern about our readiness probe for virt-handler, if the virt-handler mange to reach the API server but the watchers are not yet ready, why would we return 200 OK about our readiness?

Reply:
complete readiness is really hard to determine. Especially watchers are hard to validate from within a component. We had some attempts to do that in the past, but failed so far.